components/swap/PreviewItem.vue

```diff
@@ -22,6 +22,7 @@
     </div>
 
     <NeoButton
+      v-if="!hideRemove"
```

Comment:
where is `hideRemove` coming from?

Reply:
updated https://github.com/kodadot/nft-gallery/pull/11410/commits/563986477dc4c15ee3cdde2a28bb5fd48a3f5827